toxav/rtp.c

```diff
@@ -15,25 +15,28 @@
 
 #include "bwcontroller.h"
 
-#include "../toxcore/Messenger.h"
 #include "../toxcore/logger.h"
 #include "../toxcore/mono_time.h"
 #include "../toxcore/util.h"
+#include "../toxcore/net_crypto.h"
+#include "../toxcore/tox_private.h"
+
+Mono_Time *toxav_get_av_mono_time(ToxAV *toxav);
```

Comment:
This seems unnecessary. Is there no way to store a pointer to the `Mono_Time` in the object that needs it?

Reply:
i need to check if we need the same monotime as toxav or we can use another one ...